helm-chart/todoapp/charts/mysql/values.yaml

```diff
@@ -0,0 +1,40 @@
+namespace: mysql
+
+mysql:
+  secrets:
+    MYSQL_ROOT_PASSWORD: "1234"
+    MYSQL_USER: "app_user"
+    MYSQL_PASSWORD: "1234"
```

Comment:
Storing plaintext passwords in `values.yaml` is a security risk. Consider using a more secure method to manage sensitive data, such as using Kubernetes Secrets or a secret management tool to inject these values securely.